routes/indexRouter.js

```diff
@@ -0,0 +1,110 @@
+const routes = require('express').Router();
+
+// Middleware
+const auth = require('../middleware/auth');
+const notAuth = require('../middleware/notAuth');
+
+// Routes
+const docsRouter = require('./docsRouter');
+const loginRouter = require('./loginRouter');
+const registerRouter = require('./registerRouter');
+const dashboardRouter = require('./dashboardRouter');
+
+routes.get('/', (req, res) => {
+    res.render('index.ejs');
+})
+
+routes.get('/downloads', (req, res) => {
+    res.render('downloads.ejs');
+})
+
+routes.get('/donate', (req, res) => {
+    res.render('donate.ejs');
+})
+
+routes.get('/faq', (req, res) => {
+    res.render('faq.ejs');
+})
+
+routes.get('/knowledgebase', (req, res) => {
+    res.render('knowledgebase.ejs');
+})
+
+routes.get('/coc', (req, res) => {
+    res.render('coc.ejs');
+})
+
+routes.get('/fuchsia', (req, res) => {
+    res.render('fuchsia.ejs');
+})
+
+routes.get('/support', (req, res) => {
+    res.render('support.ejs');
+})
+
+// Docs
+
+routes.use('/docs', docsRouter);
+
+// Login/register stuff with account stuff
+
+routes.use('/login', notAuth, loginRouter);
+routes.use('/register', auth, registerRouter);
+routes.use('/dashboard', auth, dashboardRouter)
+
+routes.get('/logout', (req, res) => {
```

Comment:
does this work?

Reply:
oh wow it does! isnt that cool?!

Reply:
I don't know much about js but thats a lot of code updated

Reply:
> I don't know much about js but thats a lot of code updated

yeahhh I practically rewrote the entire website in nodejs :)